refnx/reduce/reduce.py

```diff
@@ -463,8 +463,9 @@ def _reduce_single_angle(self, scale=1):
         return deepcopy(reduction)
 
 
-def reduce_stitch(reflect_list, direct_list, norm_file_num=None,
-                  data_folder=None, trim_trailing=True, save=True, **kwds):
+def reduce_stitch(reflect_list, direct_list, background_list=None,
```

Comment:
Would it be better here to refactor reflect_list such along the lines of:
`[(708, config_dict_for_first_angle), (709, config_dict_for_second_angle)]`? Then we could get rid of direct_list and **kwds and background_list

Reply:
Eventually, something like that would make sense. Getting what we are currently using documented and committed and then later moving to config objects feels like the right approach.